src/pages/AddPage/AddPage.jsx

```diff
@@ -16,6 +19,8 @@ const AddPage = () => {
         body: JSON.stringify({ text }),
       });
       setIsSuccess(true);
+      setAllLength(allLength + 1);
+      setUncompletedLength(uncompletedLength + 1);
```

Comment:
Reset `text` so the end user can enter another task right after successful addition.

Reply:
Done.